Commander X16 Programmer's Reference Guide.md

```diff
@@ -1884,30 +1905,31 @@ The $0400-$07FF can be seen as the equivalent of $C000-$CFFF on a C64. A typical
 
 #### Zero Page
 
-|Addresses  |Description                            |
-|-----------|---------------------------------------|
-|$0000-$0001|Banking registers                      |
-|$0002-$0021|16 bit registers r0-r15 for KERNAL API |
-|$0022-$007F|Available to the user                  |
-|$0080-$009C|Used by KERNAL and DOS                 |
-|$009D-$00A8|Reserved for DOS/BASIC                 |
-|$00A9-$00D3|Used by the Math library (and BASIC)   |
-|$00D4-$00FF|Used by BASIC                          |
+Zero page is used extensively in the commander X16:  
 
-Machine code applications are free to reuse the BASIC area, and if they don't use the Math library, also that area.
+|Addresses  |Description                            |Recommendations for assembler programmers                      |
+|-----------|---------------------------------------|---------------------------------------------------------------|
+|$0000-$0001|Banking registers                      |Set ROM and RAM banks                                          |
+|$0002-$0021|16 bit registers r0-r15 for KERNAL API |Can be used, but with dependency to API calls, use with care!  |
+|$0022-$007F|Available to the user                  |Free                                                           |
+|$0080-$009C|Used by KERNAL and DOS                 |System Reserved                                                |
+|$009D-$00A8|Reserved for DOS/BASIC                 |System Reserved                                                |
+|$00A9-$00D3|Used by the Math library (and BASIC)   |Can be used if no floating point math and/or basic             |
+|$00D4-$00FF|Used by BASIC                          |Can be used if no basic                                        |
 
 #### Banking
 
 This is the allocation of banked RAM in the KERNAL/BASIC environment.
 
-|Bank |Description                                 |
-|-----|--------------------------------------------|
-|0    |Used for KERNAL/CBDOS variables and buffers |
-|1-255|Available to the user                       |
+|Bank |Description                                 |Recommendation for assembler programmers                       |
+|-----|--------------------------------------------|---------------------------------------------------------------|
+|0    |Used for KERNAL/CBDOS variables and buffers |System Reserved, **don't use!**                                |
```

Comment:
Not sure how the extra column carries any more information. It already says the first one is used by the system and the second on is available to the user.